### ISSUE TITLE
Invoice Register Mode. 

### ISSUE THREAD

**Issue by izeblood** [opensourcepos/opensourcepos#4050]
Question:  Is it possible to reflect all the payments until its fully paid (Installment payment) on the Invoice? 

![1](https://github.com/user-attachments/assets/dbbaed73-7525-494b-a15a-0d9f6e86e3f2)
![2](https://github.com/user-attachments/assets/5af8a04e-7bb6-4801-9d73-25c9d183ab99)
![3](https://github.com/user-attachments/assets/24954584-a619-4525-98ea-9745b0f03705)
![4](https://github.com/user-attachments/assets/34f57c8b-2703-4a06-b0b0-f777db10c5de)
![5](https://github.com/user-attachments/assets/7ffcaca7-4758-4dfc-85a9-37447c16abec)

**Comment by izeblood** [comment]
I need more help for this. @WebShells @andriux1990 @SteveIreland

**Comment by frooylan** [comment]
Pues ahí la llevo.... Y ahorita en la compu... Y tú? Dónde andas?

**Comment by andriux1990** [comment]
Managing installment payments with credit requires quite a bit of effort. A 'credit' payment method needs to be added, and this method should allow partial payments to be applied until the full credit amount is reached. To facilitate this, a module must be created to record and track the payments.

**Comment by andriux1990** [comment]
![Captura de pantalla_20240904_232335](https://github.com/user-attachments/assets/9a71b656-25b7-4238-89cf-b52bcefaef23)
![Captura de pantalla_20240904_232317](https://github.com/user-attachments/assets/2cf43686-305a-481d-a9fb-6e1a535ad0b7)
![Captura de pantalla_20240904_232258](https://github.com/user-attachments/assets/c5d56147-405c-41e2-83f7-02dcaf9c63cd)

**Comment by andriux1990** [comment]
An additional table should be created to record the payments made for credit sales.
![Screenshot 2024-09-04 at 23-29-59 nio-pos com _ localhost _ clinicadentalintegral _ ospos_sales_payments_tocredit phpMyAdmin 5 2 0](https://github.com/user-attachments/assets/45eb4581-96da-47d9-91a1-e9fa5f07a62d)

**Comment by izeblood** [comment]
my only concern is to not to touch anything in the modules but to put the details (value of payment) in the invoice receipt

**Comment by WebShells** [comment]
@izeblood what you are asking for was already partially done by @SteveIreland in a previous release. You can select cash and input the amount paid. Upon the next payment you will be able to register it using the same form and same way. This will record each payment along with payment date.

**Comment by izeblood** [comment]
> @izeblood what you are asking for was already partially done by @SteveIreland in a previous release. You can select cash and input the amount paid. Upon the next payment you will be able to register it using the same form and same way. This will record each payment along with payment date.

Yes...but the payments (cash) being inputted is not reflectance in the invoice receipt only the total payments...

**Comment by nukerj** [comment]
@izeblood Greetings, I'm back here after a while. Well, as @andriux1990 says, it requires a great effort and some changes in some tables of the database and create 2 new modules that are accounts receivable and another to view the payments or credits made. Precisely @andriux1990  and I spent a good time working on developing this new functionality. Although @andriux1990's version is quite advanced to what OSPOS is now in terms of design and new functionalities, we still have this adaptation for the original version of OSPOS. We are here to help if third-party service is required.

**Comment by nukerj** [comment]
Here is an example: you can enter https://45.76.0.238/mproshop user: admin password: 12345678 in the accounts receivable and payments made modules and in the sale you use the payment method Due. The server is owned by friend @andriux1990 the adaptation is from both for a specific client.

![Screenshot_20240907_231553_Chrome](https://github.com/user-attachments/assets/2396f5a5-daca-45b1-a577-954164792fe7)
![Screenshot_20240907_231610_Chrome](https://github.com/user-attachments/assets/0d16a3ab-689c-498e-8b56-e4f0812d179b)
![Screenshot_20240907_231627_Chrome](https://github.com/user-attachments/assets/cbb20ca1-a1fe-470f-bc7f-02612547eb05)
![Screenshot_20240907_231635_Chrome](https://github.com/user-attachments/assets/d91267d9-30fb-4045-a3d4-3cb85f99f65b)
![Screenshot_20240907_231700_Chrome](https://github.com/user-attachments/assets/e9d99c9b-c065-4e84-b77e-ac6d90d281e0)
![Screenshot_20240907_231712_Chrome](https://github.com/user-attachments/assets/a5b5055f-5d6e-491f-a14a-36d47eef999f)
![Screenshot_20240907_231733_Chrome](https://github.com/user-attachments/assets/2e52a945-6fd7-4269-86e1-db7b94c1b7dd)

**Comment by izeblood** [comment]
> Here is an example: you can enter https://45.76.0.238/mproshop user: admin password: 12345678 in the accounts receivable and payments made modules and in the sale you use the payment method Due. The server is owned by friend @andriux1990 the adaptation is from both for a specific client.


You worsen my query!

**Comment by izeblood** [comment]
Hey @nukerj  you worsen my query!

**Comment by nukerj** [comment]
> Hey @nukerj you worsen my query!

ok. then i delete my answer.
